docs/SPEC.md

```diff
@@ -928,7 +928,7 @@ A function produces side effects when it is explicitly declared to have side eff
 Packages are initialized in the following order:
 
 1. All imported packages are initialized and assigned to their package identifier. 
-2. All option declarations are evaluated and assigned regardless of order. An option cannot have dependencies on any other options assigned in the same package block.
+2. All option declarations are evaluated and assigned regardless of order. An option cannot have a dependency on itself or any other option assigned in the same package block.
 3. All variable declarations are evaluated and assigned regardless of order. A variable cannot have a direct or indirect dependency on itself.
```

Comment:
My understanding of the intent here is that the initialization order of variables and options should be defined by the RHSs of assignments, and not anything else (not the order of `import` statements, nor lexicographic order of package names, etc)

Reply:
Yes this is correct.